source/php/Component/Nav/nav.blade.php

```diff
@@ -38,6 +38,14 @@ class="{{$baseClass}}__link {{$item['class']}}"
                         @endif
                         @if($item['label'])
                             <span class="{{$baseClass}}__text">{{$item['label']}}</span>
+                            @if($item['children'])
```

Comment:
is the property 'children' always defined? else replace this check with @if(!empty($item['children']))